resources/leiningen/new/jcf/test/jcf/common_test.clj

```diff
@@ -0,0 +1,71 @@
+(ns {{ns}}.common-test
+  (:require byte-streams
+            [clojure
+             [spec :as s]
+             [test :refer :all]]
+            [clojure.spec.test :as spec.test]
+            [clojure.test.check
+             [clojure-test :refer [defspec]]
+             [generators :as gen]
+             [properties :as prop]]
+            [{{ns}}.common :as sut]))
+
+(use-fixtures :once (fn [f] (spec.test/instrument) (f) (spec.test/unstrument)))
```

Comment:
I think the testing stuff around `clojure.spec` has changed since I wrote this, and certainly changed in between the first 1.9 release, and this implementation. Might need to double check this approach.